__tests__/integration.test.js

```diff
@@ -124,8 +134,7 @@ describe("GET /api/articles/:article_id/comments", () => {
       .get("/api/articles/1/comments")
       .expect(200)
       .then(({ body: { comments } }) => {
-        expect(comments.length).toBe(11)
-        for (const comment of comments) {        
+        for (const comment of comments) {
           expect(comment).toEqual(
```

Comment:
We should also check the order of the comments in some way as these should be ordered as most recent first.